### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by exogenesispr**

### CODE REVIEW

File: staff/pau-julian/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# COMMANDS IN BASH
+
+Command in line of typical commands in Bash.
+
+## pwd
+```print working directory -> displays path to current directory (folder)```
```
Comment (manuelbarzi) [inline staff/pau-julian/bash/README.md]: move the explanations out from the code blocks! put them in the titles!

File: staff/pau-julian/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# COMMANDS IN BASH
+
+Command in line of typical commands in Bash.
```
Comment (manuelbarzi) [inline staff/pau-julian/bash/README.md]: proposal: change to "Typical inline commands in Bash"

File: staff/pau-julian/bash/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+## ls -l
+List file folder with details (prints a list of all visible files with attributes such as drwx)  
+```sh
+$ ls -l
+
+drwxr-xr-x  4 pc staff  128  5 feb 16:42 helloworld
+drwxr-xr-x  4 pc  staff  128  5 feb 13:42 isdi-bootcamp-202402
+```
+drwx:
```
Comment (manuelbarzi) [inline staff/pau-julian/bash/README.md]: proposal: change to "explanation to `drwx`"

File: staff/pau-julian/bash/README.md (hunk 2 above)
Comment (exogenesispr) [reply staff/pau-julian/bash/README.md]: done

File: staff/pau-julian/bash/README.md (hunk 3 above)
Comment (exogenesispr) [reply staff/pau-julian/bash/README.md]: fixed

**Pull request merged**